.azure-pipelines/client.yml

```diff
@@ -124,9 +149,16 @@ jobs:
       summaryFileLocation: '$(Build.SourcesDirectory)/coverage.xml'
       reportDirectory: '$(Build.SourcesDirectory)/htmlcov'
 
+  - task: ms.vss-governance-buildtask.governance-build-task-component-detection.ComponentGovernanceComponentDetection@0
```

Comment:
Why are we running CG from two places in the pipeline?

Reply:
No, I forgot to remove that code from the Test job. We only need to run it once under Analyze job.